modules/store-devtools/src/config.ts

```diff
@@ -1,4 +1,4 @@
-import { ActionReducer } from '@ngrx/store';
+import { ActionReducer, Action } from '@ngrx/store';
```

Comment:
Is this an unused import?

Reply:
yes and I am doing another checkin to remove those unused namespaces.
Below are required only. Basically this file does not need to be changed. 
import { InjectionToken } from '@angular/core';
import { ActionReducer } from '@ngrx/store';

Reply:
Please undo any changes to this file. This is my mistake.